src/quo/components/navigation/page_nav/view.cljs

```diff
@@ -251,86 +252,95 @@
   `:network`
     - network-name
     - network-logo a valid rn/image `:source` value"
-  [{:keys [type right-side background text-align account-switcher behind-overlay?]
+  [{:keys [type right-side background text-align account-switcher behind-overlay? center-opacity]
     :or   {type       :no-title
            text-align :center
            right-side :none
            background :white}
     :as   props}]
-  (case type
-    :no-title
-    [page-nav-base props
-     [right-content
-      {:background       background
-       :content          right-side
-       :max-actions      3
-       :behind-overlay?  behind-overlay?
-       :account-switcher account-switcher}]]
-
-    :title
-    (let [centered? (= text-align :center)]
+  (let [center-content-container-style (reanimated/apply-animations-to-style
+                                        (if center-opacity
+                                          {:opacity center-opacity}
+                                          nil)
+                                        (style/center-content-container
+                                         (if (= type :title) (= text-align :center) false)))
```

Comment:
Hey @ajayesivan 👋 Thanks for the PR 🙏 

I have some questions around this refactor, and I want to make sure I understand the code:
* I was noticing that some components like `wallet-networks-center` and `dropdown-center` were always passing `centered?` as `true`. Should they continue to do that?
* I was also noticing that most of the other components like `channel-center` always passed `centered?` as `false`. And that `title-center` was the only one that checked for `(= text-align :center)`. Should they all now only be centered based on text-align?

I suppose it makes sense that we have a standard way of determining when something should be centered, if the code had some special assumptions when always passing `false` or `true`.

Reply:
Hi @seanstrom, yes, it is a bit confusing and maybe not the ideal solution for this. But first of all, I'm not trying to fix that in this PR. My goal in this PR is to fix the animation.

In this component, only the type `title` has two alignments: `center` and `left`. So the `text-align` property is only considered when the type is `title`.

Design: https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=159-6028&m=dev